internal/cli/cli.go

```diff
@@ -159,9 +159,10 @@ func createServiceAccessCommand(controller controller.ControllerInterface) *cobr
 	return &cobra.Command{
 		Use:     "access",
 		Aliases: []string{"envs", "environment", "environments"},
-		Short:   "List all the Environment IDs and their regions",
-		Long:    `Get a list of all the Environment ID names and their regions`,
-		Args:    cobra.ExactArgs(2),
+		Short:   "Port-Forward your services to your local machine",
+		Long: `Port-Forward all services in you environment to your local machine. 
+               Requires environment ID and service ID. This commands needs to be called from within a Git repo.`,
```

Comment:
action - not tab here, otherwise u are gonna end up with something like that

```
$ kintocli help
Port-Forward all services in you environment to your local machine.
                                 Requires environment ID and service ID. This commands needs to be called from within a Git repo.
```

Reply:
my bad